apps/dashboard/src/fairness/__mock_data__/precomputedBinaryWithError.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IFairnessData } from "@responsible-ai/core-ui";
+
+import { precomputedBinary } from "./precomputedBinary";
+
+// deep copy
+const precomputedBinaryCopy = JSON.parse(JSON.stringify(precomputedBinary));
+
+Object.assign(precomputedBinaryCopy, {
+  ...precomputedBinary,
+  precomputedMetrics: [
+    // gender
+    [
+      // Model 1 metrics
+      {
+        accuracy_score: {
```

Comment:
I'm not seeing one of these with both `bounds` and `binBounds`? Or am I being blind?

Reply:
Performance metrics use bounds directly for the multi-model view, while Disparity Metrics use the binBounds to calculate the bounds for the multi-model view. binBounds are not necessary for performance metrics since they're not being displayed in the single model-view in this PR

Reply:
But shouldn't having both `bounds` and `binBounds` be valid in general?

Reply:
Yes it's valid, but not necessary